deploy/redis/templates/_helpers.tpl

```diff
@@ -57,6 +57,15 @@ Define image
 {{ .Values.image.registry | default "docker.io" }}/{{ .Values.image.repository }}:{{ .Values.image.tag }}
 {{- end }}
 
+{{- define "redis-proxy.image" -}}
+{{ .Values.redisImage.registry | default "docker.io" }}/{{ .Values.redisImage.repository }}:{{ .Values.redisImage.tag }}
+{{- end }}
+
+{{- define "busybox.image" -}}
+{{ .Values.busyboxImage.registry | default "docker.io"}}/{{ .Values.busyboxImage.repository}}:{{ .Values.redisImage.tag }}
```

Comment:
{{ .Values.redisImage.tag }} should be  {{ .Values.busyboxImage.tag }} ？

Reply:
yes, i have fixed